CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to QuantumLeap
+
+Thanks for checking out the QuantumLeap Project. We're excited to hear
+and learn from you. We've put together the following guidelines to help
+you figure out where you can best be helpful.
+
+
+### Ground rules & expectations
+
+Before we get started, here are a few things we expect from you (and
+that you should expect from others):
+
+- Be kind and thoughtful in your conversations around this project.
+  We all come from different backgrounds and projects, which means we
+  likely have different perspectives on "how open source is done."
+  Try to listen to others rather than convince them that your way is
+  correct.
+- This project is released with a [Contributor Code of Conduct][conduct].
+  By participating in this project, you agree to abide by its terms.
+- If you open a pull request, you must sign the [Individual Contributor
+  License Agreement][cla] by stating in a comment _"I have read the CLA
```

Comment:
formatting is broken

Reply:
Uh? GitHub page seems okay? i.e. markdown gets rendered properly and you can follow the links:

- https://github.com/smartsdk/ngsi-timeseries-api/blob/feature/cla/CONTRIBUTING.md